src/components/Calendar.js

```diff
@@ -1,246 +1,251 @@
+                                        day.events.length > 0 ? '!bg-secondary text-white' : 'bg-white',
                                     )}
                                 >
-                                    {day.date.split('-').pop().replace(/^0/, '')}
-                                </time>
-                                {day.events.length > 0 && (
-                                    <ol className="pl-4">
-                                        {day.events.map((event) => (
-                                            <li key={event.id} className="pb-3">
-                                                <a href={event.href} className="group flex flex-col">
-                                                    <time
-                                                        dateTime={event.datetime}
-                                                        className="hidden flex-none xl:block italic text-xs"
-                                                    >
-                                                        {event.time}
-                                                    </time>
-                                                    <p className="flex-auto font-semibold text-sm">
-                                                        {event.name}
-                                                    </p>
-                                                </a>
-                                            </li>
-                                        ))}
-                                    </ol>
-                                )}
-                            </div>
-                        ))}
-                    </div>
+                                    <time
+                                        dateTime={day.date}
+                                        className={classNames(
+                                            day.isToday
+                                                ? 'flex h-10 w-10 p-1 items-center justify-center '
+                                                + 'text-2xl border-2 rounded-full'
+                                                : 'text-2xl border-2 rounded-full border-transparent',
+                                            day.isToday && day.events.length > 0 ? 'text-white' : undefined,
+                                        )}
+                                    >
+                                        {day.date.split('-').pop().replace(/^0/, '')}
+                                    </time>
+                                    {day.events.length > 0 && (
+                                        <ol className="pl-4">
+                                            {day.events.map((event) => (
+                                                <Link to={`/${event.slug}`}>
+                                                    <li key={event.id} className="pb-3">
+                                                        <a href={event.href} className="group flex flex-col">
```

Comment:
Is this anchor neccessary, (or 2 lines above Link)?
What is difference between event.slug and event.href?
Can we use only Link component here?